code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -1223,3 +1225,15 @@
 		else
 			to_chat(user, "<span class='notice'>None of the equipment on this exosuit can use this ammo!</span>")
 	return FALSE
+
+
+/// Special light eater handling
+/obj/vehicle/sealed/mecha/proc/on_light_eater(obj/vehicle/sealed/mecha, datum/source)
```

Comment:
The mecha is the source of the signal, not the eater.
Can you fix these names?

Reply:
Sure.

Reply:
Done.